### ISSUE TITLE
Timekeeper Connect

### ISSUE THREAD

**Issue by minqii** [tillhanke/QuidditchScoreboard#31]
![grafik](https://user-images.githubusercontent.com/40029468/49882303-9ebef480-fe30-11e8-9626-f5f801a326ec.png)
 First time i tried it, it crashed. Tried again and the second time it wouldnt connect. I enter code and ID, hit connect, it puts me back in the main window but nothing happens

**Comment by tillhanke** [comment]
if it just puts you back it works,... probably (have you checked it and just tried to change score and time on timekeeper?)

**Comment by minqii** [comment]
Yes, i tried it but funnily enough it just sets the time at 00:30 (even though i already had 13 mins on the clock in timekeeper) and score wasnt updated

**Comment by minqii** [comment]
btw, in the "status" window it says "active period". dont know if thats helpful for you. seemed to me like that is somehow tied to the timekeeper server?

**Comment by tillhanke** [comment]
well it works now but is not stable at all. So this will remain open for now.
